tox.ini

```diff
@@ -16,5 +16,3 @@ deps =
     zope.minmax
     zope.publisher
     zope.testing
-
```

Comment:
I had to add the ZODB pin because it's a beta and won't install by default. `tox` is going to grab the latest releases of the dependencies listed, and since the two other version pins in `setup.py` are `>=` pins for release versions, it didn't seem to make much difference, and was another place to remember to maintain.